### ISSUE TITLE
Expose default fields for entities

### ISSUE THREAD

**Issue by gregsdennis** [gregsdennis/Manatee.Trello#74]
This stems from a client who has been receiving a "Too many checklists" error from Trello.  As it turns out they have a limit on the number of checklists that can be downloaded in a single call.

From Trello:

>I believe this user has also been in touch with us. Since their initial request's response did contain more than 5000 checklists, the solution was to specify the fields they wanted returned to not include checklists. Then, they can iterate through each of the returned cards and query for each individual cards checklist.

To support this through Manatee.Trello, the default fields for each entity need to be exposed somehow.

**Comment by gregsdennis** [comment]
As it turns out, I'm not using the `idChecklists` property data (or a lot of other data for that matter) that's being returned by Trello by default.

A possible solution for this is to just update the RestParameterRepository to specify the fields that I need.

I do like the idea of the client being able to customize the response, though.  I think that will be a later feature, continually in progress.  For now, I'll create a new issue for the problem my client was experiencing.

**Comment by gregsdennis** [comment]
See branch **feature/custom-fetch** for progress.

**Comment by gregsdennis** [comment]
Please see v1.17.1 for the short-term fix.

**Comment by gregsdennis** [comment]
This has been coded.  Waiting on a client for verification before posting.

**Comment by gregsdennis** [comment]
I have confirmed that this works well.

**Comment by gregsdennis** [comment]
Please see Manatee.Trello v1.18.0
